frontend/src/components/editor/stdin-blocking-alert.tsx

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2024 Marimo. All rights reserved. */
+
+import { Banner } from "@/plugins/impl/common/error-banner";
+import { Button } from "../ui/button";
+import { notebookAtom } from "@/core/cells/cells";
+import { atom, useAtomValue } from "jotai";
+import { Logger } from "@/utils/Logger";
+import { Kbd } from "../ui/kbd";
+import { DelayMount } from "../utils/delay-mount";
+
+// Atom to check if there is a cell with unresolved stdin
+const hasBlockingStdinAtom = atom((get) => {
+  const notebook = get(notebookAtom);
+
+  // Check each cell in order
+  for (const cellId of notebook.cellIds.inOrderIds) {
+    // Check if the cell is idle and has unresolved stdin
+    const runtime = notebook.cellRuntime[cellId];
+    if (runtime.status === "idle") {
```

Comment:
Start debugger, rerun cell in the 2000ms, and the toast never shows up.

Kinda an edgecase, but I don't think the idle check is needed at all